streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamsPartitionAssignor.java

```diff
@@ -342,6 +347,13 @@ public GroupAssignment assign(final Cluster metadata, final GroupSubscription gr
         // the maximum of the depending sub-topologies source topics' number of partitions
         final Map<Integer, TopicsInfo> topicGroups = taskManager.builder().topicGroups();
 
+        if (shutdownRequested) {
+            return new GroupAssignment(
+                    errorAssignment(clientMetadataMap,
+                            AssignorError.SHUTDOWN_REQUESTED.code())
```

Comment:
You can put all of it in one line.

```suggestion
            return new GroupAssignment(errorAssignment(clientMetadataMap, AssignorError.SHUTDOWN_REQUESTED.code()));
```

Reply:
i can